src/api/cvc4cppkind.h

```diff
@@ -2047,6 +2048,16 @@ enum CVC4_PUBLIC Kind : int32_t
    *   mkTerm(Kind kind, const std::vector<Term>& children)
    */
   BAG_COUNT,
+  /**
+   * Eliminate duplicates in a given bag. The returned bag contains exactly the
+   * same elements in the given bag, but with multiplicity one.
+   * Parameters: 1
+   *   -[1]: a term of bag sort
+   * Create with:
+   *   mkTerm(Kind kind, Term child)
+   *   mkTerm(Kind kind, const std::vector<Term>& children)
+   */
+  DUPLICATE_REMOVAL,
```

Comment:
Should we call this BAG_DUPLICATE_REMOVAL?

Reply:
`BAG_DUPLICATE_REMOVAL` is a bit long. I prefer `DUPLICATE_REMOVAL` as long as there is no confusion with other theories.

Reply:
Another alternatives I found are `squash`, `Duplicate elimination`, and recently `Deduplication`